docs/jina-ai-cloud/login.md

```diff
@@ -81,14 +81,24 @@ import hubble
 hubble.login()
 ```
 
-#### Login from Jupyter notebook/Google Colab
+#### Login from Jupyter Notebook/Google Colab
 
 ```python
 import hubble
 
 # Use Personal Access Token or browser to login.
 # Token will be saved locally.
-hubble.notebook_login()
+hubble.login()
+```
+
+#### Login from Jupyter Lab
+
+```python
+import hubble
```

Comment:
wasn't it supposed to autodetect?

Reply:
It does autodetect, but it's not 100% accurate for Jupyter Lab (there are Jupyter Lab clients that will not be detected correctly). I think it's a fair compromise to have the user specify `interactive=False`. 
For users using the standard Jupyter Lab client, it will work with `interactive=False`, so if you think we don't need to worry about that, I can remove the `interactive=False` from the documentation.

Reply:
I would suggest to have a `hint` saying this, maybe no need to have a complete subsection

Reply:
Okay that sounds great, I will see how to phrase that in the docs.